src/BugsnagUnity/Native/Android/NativePayloadClassWrapper.cs

```diff
@@ -267,7 +267,7 @@ public void SetNativeMetadataSection(string key, string section, Dictionary<stri
             return date;
         }
 
-        public void SetNativeDateTime(string key, DateTime? dateTime)
+        public void SetNativeDateTime(string key, DateTimeOffset? dateTime)
         {
             if (dateTime == null)
             {
```

Comment:
Likewise below, let's construct a `DateTimeOffset` and do the arithmetic using that.

Reply:
Done 👍